packages/gradient-block/src/components/ColorPicker.tsx

```diff
@@ -0,0 +1,131 @@
+                >
+                    <span>Configure Color</span>
+                    <span
+                        className="hover:tw-bg-box-neutral-hover hover:tw-cursor-pointer tw-rounded-sm tw-p-0.5 tw-text-strong"
+                        onClick={() => setShowColorModal(false)}
+                    >
+                        <IconCross20 />
+                    </span>
+                </div>
+            }
+            fixedFooter={
+                <div data-test-id="color-picker-flyout-footer">
+                    <FlyoutFooter
+                        buttons={[
+                            {
+                                icon: <IconCheckMark16 />,
+                                children: 'Close',
+                                onClick: () => {
+                                    if (!actualColor) {
+                                        addNewColor({
+                                            color,
+                                            position: parseInt(colorPosition),
+                                        });
+                                    } else {
+                                        color && editColor();
+                                    }
+                                    setShowColorModal(false);
+                                },
+                            },
+                        ]}
+                    />
+                </div>
+            }
+            legacyFooter={false}
+            isOpen={true}
+            contentMinHeight={195}
+            onOpenChange={() => true}
+            trigger={null}
+        >
+            <div className="tw-flex" data-test-id="color-picker-form">
+                <div className="tw-w-full tw-pt-5 tw-pl-6 tw-pr-10">
+                    <Text color="weak">Color</Text>
+                    <ColorPickerFlyout
+                        currentColor={color}
+                        onClose={() => setShowColorModal(false)}
+                        onSelect={setColor}
+                        onClick={() => {
+                            color && editColor();
+                        }}
+                    />
+                    <span className="tw-flex tw-mt-6">
+                        <Text color="weak">Stop</Text>
+                        <span className="tw-ml-1">
+                            <TooltipIcon
+                                tooltip={{
+                                    content:
+                                        'It determines when the transition from one color to the new color is complete.',
+                                }}
+                                triggerIcon={<IconQuestionMarkCircle />}
+                                iconSize={IconSize.Size12}
+                            />
+                        </span>
+                    </span>
+                    <TextInput
+                        data-test-id="test"
+                        value={colorPosition}
```

Comment:
Maybe it would make sense to round the value here, to avoid values like this:

<img width="420" alt="Bildschirm­foto 2023-03-13 um 11 05 10" src="https://user-images.githubusercontent.com/11270687/224670875-e20f59a1-ec89-4938-9a46-87f0b8bd2c87.png">

Reply:
added some validation for this Field, always integer

Reply:
Nice 🚀